include/poncos/scheduler_multi_app.hpp

```diff
@@ -1,15 +1,26 @@
 #ifndef scheduler_multi_hpp
 #define scheduler_multi_hpp
 
+#include <numeric>
+
 #include "poncos/scheduler.hpp"
 
+// per machine threshold for the membw utilization
+constexpr double PER_MACHINE_TH = 0.9;
+
```

Comment:
Is that constant used by another part of the code? if not please move it to the cpp file.